docs/getting-started-objc-macos.md

```diff
@@ -0,0 +1,151 @@
+# Getting started with macOS and Embeddinator-4000
+
+## What you will need
+
+* Follow instructions in our [Getting started with Objective-C](getting-started-objective-c.md) guide.
+
+* A .NET Assembly to use with **Embeddinator-4000**.
+
+* A macOS Cocoa Application
+
+Please continue after you have followed the instructions in our [Getting started with Objective-C](getting-started-objective-c.md) guide. If you already have a .NET Assembly you can skip directly into **Using Embeddinator-4000** section.
+
+## Creating a .NET Assembly
+
+In order to build a .NET assembly you will need to open [Visual Studio for Mac](https://www.visualstudio.com/vs/visual-studio-mac/) and create a new **.NET Library Project** by doing *File > New Solution > Other > .NET > Library* . Click next and give *Weather* as the *Project Name* and click on *Create*.
+
+Follow the next steps:
+
+1. Delete **MyClass.cs** file and **Properties** folder.
+
+2. Right click on *Weather project > Add > New File.*
+
+3. Select *Empty Class* and use **XAMWeatherFetcher** as Name and click New.
+
+4. Replace *XAMWeatherFetcher.cs* contents with the following code:
+
+```csharp
+using System;
+using System.Json;
+using System.Net;
+
+public class XAMWeatherFetcher {
+
+	static string urlTemplate = @"https://query.yahooapis.com/v1/public/yql?q=select%20item.condition%20from%20weather.forecast%20where%20woeid%20in%20(select%20woeid%20from%20geo.places(1)%20where%20text%3D%22{0}%2C%20{1}%22)&format=json&env=store%3A%2F%2Fdatatables.org%2Falltableswithkeys";
+	public string City { get; private set; }
+	public string State { get; private set; }
+
+	public XAMWeatherFetcher (string city, string state)
+	{
+		City = city;
+		State = state;
+	}
+
+	public XAMWeatherResult GetWeather ()
+	{
+		try {
+			using (var wc = new WebClient ()) {
+				var url = string.Format (urlTemplate, City, State);
+				var str = wc.DownloadString (url);
+				var json = JsonValue.Parse (str)["query"]["results"]["channel"]["item"]["condition"];
+				var result = new XAMWeatherResult (json["temp"], json["text"]);
+				return result;
+			}
+		}
+		catch (Exception ex) {
+			// Log some of the exception messages
+			Console.WriteLine (ex.Message);
+			Console.WriteLine (ex.InnerException?.Message);
+			Console.WriteLine (ex.InnerException?.InnerException?.Message);
+
+			return null;
+		}
+
+	}
+}
+
+public class XAMWeatherResult {
+	public string Temp { get; private set; }
+	public string Text { get; private set; }
+
+	public XAMWeatherResult (string temp, string text)
+	{
+		Temp = temp;
+		Text = text;
+	}
+}
+```
+
+You will notice that `Using System.Json;` is giving an error, in order to fix this we need to do the following:
```

Comment:
You will notice that `Using System.Json;` gives an error; in order to fix this we need to do the following: